js/test/Ice/exceptions/Client.ts

```diff
@@ -55,14 +55,14 @@ export class Client extends TestHelper {
                 adapter.add(new EmptyI(), Ice.stringToIdentity(""));
                 test(false);
             } catch (ex) {
-                test(ex instanceof Ice.IllegalIdentityException, ex);
+                test(ex instanceof Ice.LocalException, ex);
```

Comment:
We don't get a more specific exception?

Reply:
Maybe we can use TypeError https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError